bubble_sort/Python/bubble_sort.py

```diff
@@ -0,0 +1,14 @@
+def bubble_sort(seq):
+    changed = True
+    while changed:
+        changed = False
+        for i in range(len(seq) - 1):
+            if seq[i] > seq[i+1]:
+                seq[i], seq[i+1] = seq[i+1], seq[i]
+                changed = True
+    return seq
+
```

Comment:
The code does not comply to PEP8.

*Origin: PEP8Bear, Section: `all.autopep8`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpi7_g_6l6/bubble_sort/Python/bubble_sort.py
+++ b/tmp/tmpi7_g_6l6/bubble_sort/Python/bubble_sort.py
@@ -8,6 +8,7 @@
                 changed = True
     return seq
 
+
 if __name__ == "__main__":
     arr = [64, 34, 25, 12, 22, 11, 90]
     bubble_sort(arr)
```